package.json

```diff
@@ -24,9 +24,9 @@
   },
   "dependencies": {
     "pg": "^6.1.3",
-    "sequelize": "^3.30.2"
+    "sequelize": "3.30.2 - 5"
```

Comment:
`a - b` is an inclusive range, so I think we want `>=3.30.2 <5` since v5 might have breaking changes (see [npm's semver thing](https://semver.npmjs.com/) to test stuff out)